Mixpanel/SessionMetadata.m

```diff
@@ -15,7 +15,7 @@ - (instancetype)init {
     if (self) {
         self.eventsCounter = 0;
         self.peopleCounter = 0;
-        self.sessionID = 0;
+        _sessionID = @"";
```

Comment:
i find this a bit weird because it's not usable after `init`; you have to `reset` first, but all the other properties are okay after `init`. also, it's weird that everything else is `self.blah = ` but this is `_sessionID = `. can you make that consistent?